tests/sessions_tests/tests.py

```diff
@@ -597,33 +597,32 @@ def test_non_default_cache(self):
 
 class SessionMiddlewareTests(TestCase):
 
+    def run_middleware(self, request=None, response=None, modifier=None):
```

Comment:
That is a good question, at first I wanted to test it like this to ensure that the new middlewares are testable…